modules/administration-guide/partials/con_logging-http-traffic.adoc

```diff
@@ -0,0 +1,15 @@
+// configuring-server-logging
+
+[id="logging-http-traffic_{context}"]
+= Logging HTTP traffic
+
+It is possible to log the HTTP traffic between the {prod-short} server and the API server of the Kubernetes or OpenShift cluster.
```

Comment:
```suggestion
It is possible to log the HTTP traffic between the {prod-short} server and the API server of the {kubernetes} or OpenShift cluster.
```

Reply:
vale check is failing on `{kubernetes}`. What should be the correct attribute name?